src/object.c

```diff
@@ -1202,48 +1202,51 @@ size_t streamRadixTreeMemoryUsage(rax *rax) {
     return size;
 }
 
-/* Returns the size in bytes consumed by the key's value in RAM.
- * Note that the returned value is just an approximation, especially in the
+/* Returns the size in bytes consumed by the object header, key and value in RAM.
+ * Note that this is just an approximation, especially in the
  * case of aggregated data types where only "sample_size" elements
  * are checked and averaged to estimate the total size. */
 #define OBJ_COMPUTE_SIZE_DEF_SAMPLES 5 /* Default sample size. */
-size_t objectComputeSize(robj *key, robj *o, size_t sample_size, int dbid) {
+size_t kvobjComputeSize(robj *key, kvobj *o, size_t sample_size, int dbid) {
```

Comment:
Why use the new method name? The old feeling is still okay.

Reply:
The old method name doesn't reflect that we are evaluating only kvobj, not robj, with a key-name to take into account.